dev-tools/audius-compose

```diff
@@ -370,6 +370,7 @@ def up(
                 "docker",
                 "compose",
                 f"--project-directory={protocol_dir / 'dev-tools/compose'}",
+                f"--project-name={protocol_dir.name}",
```

Comment:
@raymondjacobson your cleanup moved docker-compose.yml from "audius-protocol" dir to "compose" dir, which also changed the project name. this line just reverts the project name to the status quo since the dev nginx ingress expects the containers to have that `audius-protocol-` prefix

Reply:
oh wow great catch. sorry about that.